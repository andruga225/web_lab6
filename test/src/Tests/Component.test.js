import renderer from 'react-test-renderer';
import {Div, P, Label} from './Component'

it('Div', () => {
    const result = renderer.create(
        <Div params={"Hello"}></Div>
    );
    expect(result).toMatchSnapshot();
})

it('P', () => {
    const result = renderer.create(
        <P params={"Another hello!"}/>
    );
    expect(result).toMatchSnapshot();
})

it('Label', () => {
    const result = renderer.create(
        <Label params={"Label"}/>
    );
    expect(result).toMatchSnapshot();
})