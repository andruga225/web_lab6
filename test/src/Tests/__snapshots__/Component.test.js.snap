// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Div 1`] = `<div />`;

exports[`Label 1`] = `<label />`;

exports[`P 1`] = `<p />`;
